base_layer/core/src/proof_of_work/error.rs

```diff
@@ -48,7 +48,9 @@ pub enum DifficultyAdjustmentError {
 }
 
 #[derive(Debug, Error)]
-pub enum DifficultyError {
+pub enum LwmaError {
```

Comment:
This does not seem like a good renaming. The variant "InvalidDifficulty" does not seem related to Lwma at all